WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewController.swift

```diff
@@ -591,6 +591,7 @@ private extension ShippingLabelFormViewController {
     /// Present a Top Banner View containing the EU Shipping Notice.
     ///
     func showTopBannerView() {
+        ServiceLocator.analytics.track(.euShippingNoticeShown)
```

Comment:
Tip: We prefer to create analytic events instead as they are easier to extend in the future.

Reply:
Good point! Adjusted in https://github.com/woocommerce/woocommerce-ios/pull/9766/commits/568d2d5165ff859888e0bbaf0358f3e0ba928c9a and https://github.com/woocommerce/woocommerce-ios/pull/9766/commits/d96cefd2c86dc8f7fc66ee13d975e5cc9542541f